lucid/optvis/objectives.py

```diff
@@ -61,41 +61,30 @@ class Objective(object):
 
   def __init__(self, objective_func, name="", description=""):
     self.objective_func = objective_func
-    self.name = name
     self.description = description
+    self.value = None # This value is populated after a call
```

Comment:
Let's briefly think through whether to call this `tensor` instead. When we say "value", we often mean a concrete Python value, not a Tensor in a graph.

Reply:
Test